src/ComposerChecksPlugin.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace Lullabot;
+namespace ComposerChecks;
```

Comment:
This should be `Lullabot\ComposerChecks` I think?

Reply:
Ups! Yes. Updating it...